perftracker/templates/comparison_id.html

```diff
@@ -27,15 +27,45 @@ <h4>Job: {{ job }}</h4>
     </div>
   </div>
 </div>
+
 <span style="float: right">
-  <a onclick="show_all_tables()">Show tables</a>
-  <a onclick="hide_all_tables()">Hide tables</a>
   <a id="analyze_button" onclick="pt_charts_analyze()">Analyze</a>
   <a onclick="pt_label_data()">Label data</a>
 </span>
 
 <div id="main_content_container"></div>
 
+<button class="panel__btn_toggle"><i class="fa fa-angle-up"></i></button>
+<nav id="panel_id" class="navbar-default navbar-fixed-bottom panel">
+  <div class="panel__button-controller panel__unit_hide-tables">
+      <h4><button onclick="hide_all_tables()">Hide tables</button></h4>
+  </div>
+  <div class="panel__button-controller panel__unit_show-tables">
+      <h4><button onclick="show_all_tables()">Show tables</button></h4>
+  </div>
+  <div class="panel__button-controller panel__unit_unselect">
+      <h4><button onclick="unselectGraphs()">Unselect All</button></h4>
+  </div>
+  <div class="panel__button-controller panel__unit_show-selected">
+      <h4><button onclick="showSelectedGraphs()">Show Selected</button></h4>
+  </div>
+  <div class="panel__button-controller panel__unit_go-to">
+      <h4><button>Group...</button></h4>
+      <div class="panel_dropup-content">
+          {% for g_tag, g in cmp_view.groups.items %}
```

Comment:
don't use django templates when possible, they generate too much HTML-code, that can be avoided